charts/gatekeeper-operator/crds/constrainttemplatepodstatus-customresourcedefinition.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.3.0
+    helm.sh/hook: crd-install
+    helm.sh/hook-delete-policy: before-hook-creation
```

Comment:
I don't think this applies to helm 3 anymore, so please can we remove such hooks and make this a nice chart?

Reply:
ok done